CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 ## 3.39.0 (Unreleased)
 
+IMPROVEMENTS:
+* compute: added network field to `compute_target_instance` ([#2456](https://github.com/hashicorp/terraform-provider-google-beta/pull/2456))
+* compute: added storage_locations field to `google_compute_snapshot` ([#2461](https://github.com/hashicorp/terraform-provider-google-beta/pull/2461))
+* compute: added `kms_key_service_account`, `kms_key_self_link ` fields to `snapshot_encryption_key` field in `google_compute_snapshot`
+* compute: added `source_disk_encryption_key.kms_key_service_account` field to `google_compute_snapshot`
```

Comment:
These two should have the same PR link as the top one I believe.

Reply:
Ah yep- I'd added the links to the original PR then copied them here. added in both tpg/tpgb.